tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -592,6 +592,11 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         if (activeDashboards && selectedDashboard == null) {
           selectedDashboard = activeDashboards[0] || null;
           if (selectedDashboard != null) {
+            // Use location.replace for this call to avoid breaking back-button navigation.
+            // Note that this will precede the update to tf_storage triggered by updating
+            // _selectedDashboard.
```

Comment:
nit: maybe just add "... and make it a no-op" to the end of this comment for maximum clarity.